src/main/java/com/feedzai/commons/sql/abstraction/util/Constants.java

```diff
@@ -84,4 +84,14 @@
      * server. This is only possible on implementations that support this behavior. The default value is {@code false}.
      */
     public static final boolean DEFAULT_DISABLE_LOB_CACHING = false;
+
+    /**
+     * Default duration (in seconds) to wait for the database connection to be established.
+     */
+    public static final String DEFAULT_LOGIN_TIMEOUT = "0";
```

Comment:
Explain in the javadoc why the default is 0 and what it means. I'm guessing it means there is no timeout for most databases.

Reply:
Yes, you are right, 0 means no timeout. I will explain it there. Thanks for the feedback.